logging/log_config.go

```diff
@@ -13,17 +13,46 @@ type LoggerPatternConfig struct {
 }
 
 const (
+	// pattern matching on loggers.
 	validLoggerSectionName             = `[a-zA-Z0-9]+([_-]*[a-zA-Z0-9]+)*`
 	validLoggerSectionNameWithWildcard = `(` + validLoggerSectionName + `|\*)`
 	validLoggerSections                = validLoggerSectionName + `(\.` + validLoggerSectionName + `)*`
 	validLoggerSectionsWithWildcard    = validLoggerSectionNameWithWildcard + `(\.` + validLoggerSectionNameWithWildcard + `)*`
 	validLoggerName                    = `^` + validLoggerSectionsWithWildcard + `$`
+
+	// resource configurations.
+	validNamespacePattern       = `([\w-]+|\*)`
+	validResourceTypePattern    = `(service|component|\*)`
```

Comment:
Remotes aren't included in this? Or do those (`rdk:remote:/foo`) fall under the `loggerPatternRegexp`?

Reply:
Included it under `validTypeSubsectionPattern`. The `resourcePatternRegexp` will be able to validate `rdk:remote:/foo`